sql/moz-fx-data-shared-prod/firefox_ios_derived/firefox_ios_clients_v1/checks.sql

```diff
@@ -1,9 +1,9 @@
 #fail
 {{ is_unique(columns=["client_id"]) }}
 #fail
-{{ not_null(columns=["client_id"]) }}
+{{ not_null(columns=["client_id"], where="first_seen_date = @submission_date") }}
 #fail
-{{ min_row_count(1, "first_seen_date = @submission_date") }}
+{{ min_row_count(1, where="first_seen_date = @submission_date") }}
```

Comment:
Was the SQL generated before this change correct?

Reply:
yes, I noticed in the next check jinja error would arise when omitting the `where` so decided to add it to this one explicitly too.